src/Writers/HTMLWriter.jl

```diff
@@ -1030,7 +1030,11 @@ mdconvert(h::Markdown.Header{N}, parent; kwargs...) where {N} = DOM.Tag(Symbol("
 
 mdconvert(::Markdown.HorizontalRule, parent; kwargs...) = Tag(:hr)()
 
-mdconvert(i::Markdown.Image, parent; kwargs...) = Tag(:img)[:src => i.url, :alt => i.alt]
+mdconvert(i::Markdown.Image, parent; kwargs...) = if endswith(i.url, ".webm")
+    Tag(:video)[:src => i.url, :width => "800", :controls => "true", Symbol("data-title") => i.alt]
```

Comment:
Not sure we really have browsers out there that do not support `<video>` anymore, but I guess it wouldn't hurt to have fallback content in the tag too, say with a link to the file?